### DISCUSSION TITLE
Visualizing atom level property

### DISCUSSION THREAD

**Discussion by SinaSarparast** [molstar/molstar#601] [Q&A]
Hello,

I want to project an atom level property (b factor) on the protein structures in form of a heat map (see the image). Is it possible to this with this package? Any help on how I can visualize this value on the protein structure is highly appreciated. 

![image](https://user-images.githubusercontent.com/26283630/198314721-97e86d4f-2f72-470f-a3cb-0c56cd2aa6bb.png)

Thanks

**Comment by dsehnal** [comment]
hi, yes, you can do that with the Uncertainty/Disorder theme.

![image](https://user-images.githubusercontent.com/21145776/198400838-e5f7937f-1a72-4ef9-b870-4414207e4f91.png)
